web/src/features/fba/components/map/featureStylers.ts

```diff
@@ -150,9 +150,29 @@ export const getAdvisoryFillColor = (fireShapeStatus: FireShapeStatus) => {
   }
 }
 
+/**
+ * Given an OpenLayers feature from the fire zone unit label layer, return a label to display on the map.
+ * @param feature The feature of interest from the fire zone unit layer.
+ * @returns A string to be used as a label on the map.
+ */
+const getFireZoneUnitLabel = (feature: RenderFeature | ol.Feature<Geometry>) => {
+  const fireZoneId = feature.getProperties().FIRE_ZONE_
+  let fireZoneUnit = feature.getProperties().FIRE_ZON_1
```

Comment:
Are these properties spelled correctly?

Reply:
Yep, property names are correct. Property names are longer (more descriptive) in the original shapefile, but to create the underlying pmtiles layer we first had to create a geojson layer which has a limit on property name length and truncates them.